doc/tutorial/index.rst

```diff
@@ -10,5 +10,4 @@ Tutorial
     session
     repos
     queries
-    depsolve
```

Comment:
@dmach I wasn't entirely sure what was your plan with the depsolve tutorial, for now I've put the whole thing into `transaction`. I can still split it up if it'd make sense...

Reply:
@lukash IIRC I wanted to document how to resolve dependencies, not necessarily run the transaction (that would be part of another step). But both cases are probably are quite close to each other, I suppose.
Maybe look at this from user perspective - what they want to do is to resolve deps (and that should be the wording in documentation IMO). Libdnf makes it it possible via populating and resolving a transaction.

Reply:
Ok, I've renamed the tutorial to indicate it covers both resolving dependencies and running a transaction but I've kept it as a single piece of code for now. The reason being if split, the running a transaction tutorial would be having a nontrivial and tightly coupled setup (the depsolving) on a different page, so I'd need to add some more explanation text for that so that it's not confusing (I've already added similar text for explaining the base and repo setups are on different pages) and it seems it would do more harm than good.